### PR TITLE
Add unit tests for the DataX.Flow service

### PR DESCRIPTION
**Pull request by kjcho-msft**

### CODE REVIEW

File: Services/DataX.Flow/DataX.Flow.SchemaInference/Kafka/KafkaMessageBus.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 namespace DataX.Flow.SchemaInference.Kafka
 {
-    public class KafkaMessageBus : IMessageBus
+    public class KafkaMessageBus : IMessageBus, IDisposable
```
Comment (carlbrochu) [inline Services/DataX.Flow/DataX.Flow.SchemaInference/Kafka/KafkaMessageBus.cs]: I assume these are good things to be implementing, were these reported anywhere?

File: Services/DataX.Flow/DataX.Flow.Common/Helper.cs [hunk 2]
```diff
@@ -309,20 +308,8 @@ public static string PathResolver(string path)
...
-                Regex r = new Regex(@"^((keyvault|secretscope:?):\/\/)?([^:\/\s]+)(\/)(.*)?", RegexOptions.IgnoreCase);
-                var keyvault = string.Empty;
-
-                var secret = string.Empty;
-                MatchCollection match = r.Matches(path);
-                if (match != null && match.Count > 0)
-                {
-                    foreach (Match m in match)
-                    {
-                        keyvault = m.Groups[3].Value.Trim();
```
Comment (carlbrochu) [inline Services/DataX.Flow/DataX.Flow.Common/Helper.cs]: great to see this refactored too

File: Services/DataX.Flow/DataX.Flow.SchemaInference/Kafka/KafkaMessageBus.cs (hunk 1 above)
Comment (kjcho-msft) [reply Services/DataX.Flow/DataX.Flow.SchemaInference/Kafka/KafkaMessageBus.cs]: The roslyn analyzer detects this and displays a message in the output/ error list panel.

File: Services/DataX.Flow/DataX.Flow.Common/Helper.cs (hunk 2 above)
Comment (kjcho-msft) [reply Services/DataX.Flow/DataX.Flow.Common/Helper.cs]: thanks!

**Pull request merged**